ocs_ci/ocs/ui/base_ui.py

```diff
@@ -1027,7 +1027,8 @@ def close_browser(driver):
     logger.info("Close browser")
     take_screenshot(driver)
     copy_dom(driver)
-    driver.close()
+    driver.quit()
+    time.sleep(10)
```

Comment:
not critical, but I would add log.info('closing Selenium browser, sleep 10s'), because we may seek then in logs what triggered another 10 sec of doing nothing.